src/commands/dev/convert/messages.ts

```diff
@@ -40,29 +54,43 @@ export default class DevConvertMessages extends SfCommand<DevConvertMessagesResu
 
   public async run(): Promise<DevConvertMessagesResult[]> {
     const { flags } = await this.parse(DevConvertMessages);
-
+    const projectDir = resolve(flags['project-dir']);
+    const { name } = JSON.parse(await fs.promises.readFile(resolve(projectDir, 'package.json'), 'utf8')) as {
```

Comment:
Maybe helpful to make this more descriptive since there are already a bunch of other variables with the word "name" in them flying around here.
```suggestion
    const { name as pluginName } = JSON.parse(await fs.promises.readFile(resolve(projectDir, 'package.json'), 'utf8')) as {
```